charts/swoop-caboose/README.md

```diff
@@ -132,6 +132,160 @@ time="2023-07-28T17:45:34Z" level=info msg="index config" indexWorkflowSemaphore
+
+### Running Argo Workflows Copy Assets Stac Task
+The [copy-assets-stac-task](https://github.com/Element84/copy-assets-stac-task) example will run an argo workflow which copies specified Assets from Source STAC Item(s), uploads them to S3 and updates Item assets hrefs to point to the new location.
+
+Now, prior to running the argo workflows example, first make sure to port-forward `minio` onto localhost ports `9000` & `9001`.
+
+Via Rancher Desktop:
+<br></br>
+<p align="center">
+  <img src="../../images/minio-port-forwarding-bundle.png" alt="Port forwarding Minio" width="1776">
+</p>
+<br></br>
+
+or via terminal:
+
+```
+kubectl port-forward svc/minio 9000:9000 &
+kubectl port-forward svc/minio 9001:9001 &
+```
+
+#### Install First the MinIO client by running
+
+```
+brew install minio/stable/mc
+```
+
+#### Then set the MinIO alias, find the ACCESS_KEY and SECRET_KEY by quering the Helm values
+
+```
+export MINIO_ACCESS_KEY=`helm get values minio -a -o json | jq -r .service.accessKeyId | base64 --decode`
+export MINIO_SECRET_KEY=`helm get values minio -a -o json | jq -r .service.secretAccessKey | base64 --decode`
+mc alias set swoopminio http://127.0.0.1:9000 $MINIO_ACCESS_KEY $MINIO_SECRET_KEY
+```
+
+####  Run the [copy-assets-stac-task](https://github.com/Element84/copy-assets-stac-task) argo workflow example
+
+First clone the [copy-assets-stac-task](https://github.com/Element84/copy-assets-stac-task) repository.
+
+After cloning the [copy-assets-stac-task](https://github.com/Element84/copy-assets-stac-task) repository, first proceed to modify the `payload_workflow.json` and replace the `<REPLACE_WITH_ASSETS_S3_BUCKET_NAME>` with the bucket name created on the [Pre-requisites section](#pre-requisites-to-running-argo-workflows-example).
+
+Create a public minio local bucket and copy the `payload_workflow.json` file after replacing the `<REPLACE_WITH_ASSETS_S3_BUCKET_NAME>` name. To do this via the minio client:
+
+```
+$ mc mb swoopminio/payloadtest
+
+Bucket created successfully `swoopminio/payloadtest`.
+```
+```
+$ mc anonymous set public swoopminio/payloadtest
+
+Access permission for `swoopminio/payloadtest` is set to `public`
+```
+```
+$ mc cp ./payload_workflow.json swoopminio/payloadtest/
+
+...opy-assets-stac-task/payload_workflow.json: 4.58 KiB / 4.58 KiB ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 127.61 KiB/s 0s
+```
+
+Now modify the `secrets.yaml` and replace the `<REPLACE_WITH_BASE64_AWS_ACCESS_KEY>`, `<REPLACE_WITH_BASE64_AWS_SECRET_ACCESS_KEY>` and `<REPLACE_WITH_BASE64_AWS_REGION>` with the base64 encoded version of the AWS_ACCESS_KEY, AWS_SECRET_ACCESS_KEY and AWS_REGION of the iam user and bucket created on the [Pre-requisites section](#pre-requisites-to-running-argo-workflows-example).
+
+Create the kubernetes secret by executing:
+```
+kubectl apply -n default -f ./secret.yaml
```

Comment:
should this secret be called `copy-stac-asset-s3-secret` instead as it contains credentials for S3 and not MinIO?

Reply:
Fixed.